main/res/values/strings.xml

```diff
@@ -662,9 +662,9 @@
     <string name="init_backup_success">c:geo\'s database was successfully copied to:</string>
     <string name="init_backup_failed">Backup of c:geo\'s database failed.</string>
     <string name="init_backup_unnecessary">Database is empty, no backup necessary.</string>
-    <string name="init_backup_folder_exists_error">Couldn\'t create a new backup directory. Have you already made a backup in the last minute?</string>
+    <string name="init_backup_folder_exists_error">Couldn\'t create a new backup folder. Have you already made a backup in the last minute?</string>
```

Comment:
How about `No new backup folder could be created. Have you already created a backup within the last minute?`

Reply:
How about `Unable to create the new backup folder. Have you ...`?